inference-engine/src/transformations/src/transformations/common_optimizations/hswish_fusion.cpp

```diff
@@ -219,3 +219,48 @@ ngraph::pass::HSwishFusionWithClamp::HSwishFusionWithClamp() {
     auto m = std::make_shared<ngraph::pattern::Matcher>(mul_second, matcher_name);
     register_matcher(m, callback);
 }
+
+NGRAPH_RTTI_DEFINITION(ngraph::pass::HSwishFusionWithClampDiv, "HSwishFusionWithClampDiv", 0);
+
+ngraph::pass::HSwishFusionWithClampDiv::HSwishFusionWithClampDiv() {
+    MATCHER_SCOPE(HSwishFusionWithClampDiv);
+    // Replaces a sub-graph x * (Clamp(x + 3, 0, 6) / 6) with a HSwish op.
+    auto input = ngraph::pattern::any_input();
+    auto add_constant = ngraph::pattern::wrap_type<ngraph::opset4::Constant>();
+    auto add = std::make_shared<ngraph::opset4::Add>(input, add_constant);
+    auto clamp = std::make_shared<ngraph::op::v0::Clamp>(add, 0.0f, 6.0f);
+    auto div_constant = ngraph::pattern::wrap_type<ngraph::opset4::Constant>();
+    auto div = std::make_shared<ngraph::opset4::Divide>(clamp, div_constant);
+    auto mul = std::make_shared<ngraph::opset4::Multiply>(input, div);
+
+    ngraph::matcher_pass_callback callback = [=](ngraph::pattern::Matcher &m) {
+        auto &pattern_to_output = m.get_pattern_value_map();
+        auto x_output = pattern_to_output.at(input);
+
+        auto add_const_value = std::dynamic_pointer_cast<ngraph::opset4::Constant>(pattern_to_output.at(add_constant).get_node_shared_ptr());
+        auto div_const_value = std::dynamic_pointer_cast<ngraph::opset4::Constant>(pattern_to_output.at(div_constant).get_node_shared_ptr());
+
+        bool valid_constant_values = op::util::has_constant_value(add_const_value, 3.0)
+                                     && op::util::has_constant_value(div_const_value, 6.0);
+        if (!valid_constant_values) {
+            return false;
+        }
+
+        auto hswish = std::make_shared<ngraph::opset4::HSwish>(x_output);
+
+        hswish->set_friendly_name(m.get_match_root()->get_friendly_name());
+        ngraph::copy_runtime_info({ pattern_to_output.at(add_constant).get_node_shared_ptr(),
+                                    pattern_to_output.at(add).get_node_shared_ptr(),
+                                    pattern_to_output.at(clamp).get_node_shared_ptr(),
+                                    pattern_to_output.at(div_constant).get_node_shared_ptr(),
+                                    pattern_to_output.at(div).get_node_shared_ptr(),
+                                    pattern_to_output.at(mul).get_node_shared_ptr()
+                                  },
+                                  hswish);
```

Comment:
You don't need to copy rt info for constants.
```suggestion
        ngraph::copy_runtime_info({pattern_to_output.at(add).get_node_shared_ptr(),
                                    pattern_to_output.at(clamp).get_node_shared_ptr(),
                                    pattern_to_output.at(div).get_node_shared_ptr(),
                                    pattern_to_output.at(mul).get_node_shared_ptr()
                                  },
                                  hswish);
```